README.md

```diff
@@ -77,6 +77,11 @@ The action works without configuration, however you can provide options for cust
           # merge commit, and it's easy to commit this by mistake. Enable this option
           # to also validate the commit message for one commit PRs.
           validateSingleCommit: true
+          # When using "validateSingleCommit", even if you edit the PR title, the PR
+          # title is not used for the merge commit. Also, since it cannot detect an error
+          # as long as the commit message is semantic, it's easy to commit this by mistake.
+          # Enable this option to also validate if the PR title matches the commit title.
```

Comment:
Maybe we can shorten this a bit by referencing the `validateSingleCommit` option? Seems like most of the relevant information is already there.

```suggestion
          # Related to `validateSingleCommit` you can opt-in to validate that the PR
          # title matches a single commit to avoid confusion.
```

What do you think?

Reply:
Thank you!
Hmm, since we're familiar with this problem, it's fine for us.
But for people who don't know well, I'm not sure this is enough.

However, I think we can add more descriptions if someone asks, so I feel this is okay for now! I'll apply this.

Reply:
Do you think there's something missing in the description for `validateSingleCommit`?

Btw. can you update the description in `action.yml` to match this? It's unfortunately a bit a hassle to sync these.

Reply:
Applied in c101b573d402bd412b28b48532719bf970bfe6f5.